recipes/aicspylibczi/build.sh

```diff
@@ -1,2 +1,2 @@
 export CMAKE_GENERATOR=Ninja
-{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+python3 -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
```suggestion
${PYTHON} -m pip install . -vv --no-deps --no-build-isolation
```

Reply:
You are right that you cannot use jinja templating. But you can (and should) use some environment variables (c.f. https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html).